chromadb/db/mixins/embeddings_queue.py

```diff
@@ -288,7 +288,9 @@ def _backfill(self, subscription: Subscription) -> None:
             .where(t.topic == ParameterValue(subscription.topic_name))
             .where(t.seq_id > ParameterValue(subscription.start))
             .where(t.seq_id <= ParameterValue(subscription.end))
-            .select(t.seq_id, t.operation, t.id, t.vector, t.encoding, t.metadata)
+            .select(
+                t.seq_id, t.operation, t.id, t.vector, t.encoding, t.metadata, t.topic
```

Comment:
I don't think `topic` is needed here. Perhaps wrongfully added during testing/experimentation